### DISCUSSION TITLE
Copy local data from local to docker

### DISCUSSION THREAD

**Discussion by Ellysian** [danny-avila/LibreChat#5140] [Q&A]
I have been using LibreChat with npm for a while now. I have all my settings, presets, agents, and chats there. I recently switched to running LibreChat with Docker, and everything works great. Except all my custom settings are no longer there, and as if I just installed a fresh copy.

Is there a simple way to copy all the local data from local installation (that I run with npm) to docker container?

**Comment by danny-avila** [comment]
if you were running on npm, you were likely connecting to MongoDB installed on your machine or using Atlas.

Use the override file to change the MONGO_URI env variable if you want to point to the same Mongo connection you were using before:
https://www.librechat.ai/docs/configuration/docker_override

example:

```yaml
services:
  api:
    environment:
        - MONGO_URI=your-mongo-connection-string
```

If you want to keep everything as is, and simply port in your data, you will need to look into mongoexport but this is just a tad bit more advanced
